rules/integrations/aws/ml_cloudtrail_rare_method_by_city.toml

```diff
@@ -23,7 +24,12 @@ interval = "15m"
 license = "Elastic License v2"
 machine_learning_job_id = "rare_method_for_a_city"
 name = "Unusual City For an AWS Command"
-note = """## Triage and analysis
+note = """
+## Config
```

Comment:
@brokensound77 you okay that I added these notes for the  cloudtrail ML jobs and moved them to the AWS integration?

Reply:
💯 on the config note.

As far as moving them - we really have not done much with their sub dirs to this point (short of any unit tests using them), but since we can always detect ML rules by `type`, we are ok. 

The rest of the directories are multi rule type anyway, categorized by their focused datasource, so that is more inline. It may make the remaining ML folder sort of out of place then and I wonder if we just integrate those in to where they most apply. I am fine either way